test/resources/immutable-items-strict-doc-definitions.js

```diff
@@ -0,0 +1,43 @@
+function() {
```

Comment:
As noted in the pull request description, this file borrows from `immutable-items-doc-definitions.js`.